transform-plugins/widgets/ValueMapper-transform.json

```diff
@@ -5,12 +5,18 @@
   "configuration-groups": [
     {
       "properties": [
-        {
-          "widget-type": "textbox",
-          "label": "mapping",
-          "name": "mapping"
-        },
-        {
+       {
+       	  "name": "mapping",
+	  "label": "mapping",
+	  "widget-type": "ds-multiplevalues",
+	  "widget-attributes": {
+	  "delimiter": ",",
```

Comment:
should indent lines 13-17, since they're a part of the 'widget-attributes' field